gnc/navigator_perception/helper/model.cc

```diff
@@ -0,0 +1,9 @@
+#include <navigator_vision_lib/model.h>
+
+Model::Model(float width, float height): width(width), height(height)
```

Comment:
Do constructor initializers like this: [(from the google style guide)](https://google.github.io/styleguide/cppguide.html#Constructor_Initializer_Lists)
// When the list spans multiple lines, put each member on its own line
// and align them:
MyClass::MyClass(int var)
    : some_var_(var),             // 4 space indent
      some_other_var_(var + 1) {  // lined up
  DoSomething();
}